babel.config.js

```diff
@@ -0,0 +1,5 @@
+'use strict'
+
+module.exports = {
+  presets: ['@babel/preset-env']
+}
```

Comment:
Are there something different from babelrc file?

Reply:
It was created when i ran:
```bash
vue add babel # vue is @vue/cli
```
inside the project. It is different in sense its a future proof solution that forces all dependencies to be transpiled together with your code: https://babeljs.io/docs/en/configuration#whats-your-use-case